tests/components/deconz/test_sensor.py

```diff
@@ -165,11 +155,7 @@ async def test_sensors(hass):
     gateway.api.event_handler(state_changed_event)
     await hass.async_block_till_done()
 
-    light_level_sensor = hass.states.get("sensor.light_level_sensor")
-    assert light_level_sensor.state == "1.6"
-
-    switch_2_battery_level = hass.states.get("sensor.switch_2_battery_level")
-    assert switch_2_battery_level.state == "75"
+    assert hass.states.get("sensor.switch_2_battery_level").state == "75"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)